src/utils/url.ts

```diff
@@ -24,14 +27,15 @@ const getTransactionIdFromRequestParams = (req: Request): string => req.params[c
 const getSubmissionIdFromRequestParams = (req: Request): string => req.params[config.ROUTE_PARAM_SUBMISSION_ID];
 
 export const isRemoveJourney = (req: Request): boolean => {
-  return req.query[config.JOURNEY_QUERY_PARAM] === config.JourneyType.remove;
-};
+  const session = req.session as Session;
```

Comment:
Need to add some tests around this new implementation. Either now or after merging into @mattch1 ' s PR.

Reply:
Tests added.